scenarios/triso/deployment_calcs/existing_lwr_expansion.py

```diff
@@ -0,0 +1,197 @@
+                <n_assem_core>
+                    {attributes['n_assem_core']}
+                </n_assem_core>
+                <n_assem_batch>
+                    {attributes['n_assem_batch']}
+                </n_assem_batch>
+                <power_cap>{attributes['power_cap']}</power_cap>
+            </Reactor>
+        </config>
+    </facility>
+    """).strip()
+
+
+def write_xml_to_file(output_dir, state, number, reactor_type, xml_string):
+    """
+    Write the XML string to a file.
+
+    Parameters
+    ----------
+    output_dir : str
+        The directory to output the XML files to.
+    state : str
+        The state where the reactor is located.
+    number : int
+        The reactor number.
+    reactor_type : str
+        The type of reactor ('large' or 'small').
+    xml_string : str
+        The generated XML string.
+
+    Returns
+    -------
+    None
+    """
+    file_path = os.path.join(
+                              output_dir,
+                              f"{state}_{number}_{reactor_type}_est.xml")
+    with open(file_path, 'w') as f:
+        f.write(xml_string)
+
+
+def generate_est_facility_xml(df, output_dir, reactor_types):
+    """
+    Generate the XML string for the estimated facilities from
+    the given dataframe.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        The dataframe containing the information about the reactors.
+    output_dir : str
+        The directory to output the XML files to.
+    reactor_types : dict
+        The dictionary containing the reactor types and their attributes.
+
+    Returns
+    -------
+    None
+    """
+    create_output_directory(output_dir)
+
+    for state in range(1, len(df['Plant state'])):
+        st = df.loc[state, 'Plant state']
+        num_large = df.loc[state, 'Number_sites_space_large']
+        num_small = df.loc[state, 'Number_sites_space_small']
+
+        # Write the XML files for each large reactor in this state
+        for number in range(int(num_large)):
+            xml_string = generate_xml_string(
+                                              st, number, 'large',
+                                              reactor_types['large'])
+            write_xml_to_file(output_dir, st, number, 'large', xml_string)
+
+        # Write the XML files for each small reactor in this state
+        for number in range(int(num_small)):
+            xml_string = generate_xml_string(
+                                              st, number, 'small',
+                                              reactor_types['small'])
+            write_xml_to_file(output_dir, st, number, 'small', xml_string)
+
+    print('Successfully generated reactor XML files.')
+
+
+# Example usage
+if __name__ == "__main__":
+    import camelot
+    import pandas as pd
+
+    base = 'https://fuelcycleoptions.inl.gov/SiteAssets/SitePages/Home/'
+    stem = 'Evaluation%20of%20NPP%20and%20CPP%20Sites%20Aug%2016%202024.pdf'
+    report_url = base + stem
+
```

Comment:
'base' and 'stem' are hard-coded here - will they always be the same?  And could you simply hard-code the report-url instead?